benches/osrank_naive.rs

```diff
@@ -58,15 +60,44 @@ fn construct_network_small() -> Network<f64> {
 }
 
 fn construct_network() -> Network<f64> {
-    let deps_csv_file = File::open("data/cargo_dependencies.csv").unwrap();
-    let deps_meta_csv_file = File::open("data/cargo_dependencies_meta_bench_sample.csv").unwrap();
-    let contribs_csv_file = File::open("data/cargo_contributions_bench_sample.csv").unwrap();
+    let deps_reader = BufReader::new(File::open("data/cargo_dependencies.csv").unwrap())
+        .split(b'\n')
+        .map(|l| l.unwrap())
+        .fold(Vec::default(), |mut acc, v| {
+            acc.extend(v);
+            acc.push(b'\n');
+            acc
+        });
+
+    let deps_meta_reader = BufReader::new(File::open("data/cargo_dependencies_meta.csv").unwrap())
+        .split(b'\n')
+        .map(|l| l.unwrap())
+        .take(1000)
+        .fold(Vec::default(), |mut acc, v| {
+            acc.extend(v);
+            acc.push(b'\n');
```

Comment:
We need to re-add the `\n` that `split` has removed, before flattening everything into a `Vec<u8>`.